test/OpenTelemetry.Instrumentation.AspNet.Tests/HttpInMetricsListenerTests.cs

```diff
@@ -24,6 +24,7 @@
 
 namespace OpenTelemetry.Instrumentation.AspNet.Tests;
 
+[Collection("HttpInListenerTests")]
```

Comment:
Could we use `[assembly: CollectionBehavior(DisableTestParallelization = true)]` instead? This would run all the tests from this assembly sequentially, but this is more intuitive.

For reference: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/test/OpenTelemetry.Contrib.Extensions.AWSXRay.Tests/AssemblyInfo.cs#L19

Reply:
Sure, works on my local machine.

Fixed in 7a7069bc6762d8e9ecec92a6ce2ba5b23d5163c7.